blog/_src/posts/2016-11-16-constructive-galois-connections.md

```diff
@@ -0,0 +1,167 @@
+    Title: Understanding Constructive Galois Connections
+    Date: 2016-11-16T00:00:00
+    Tags: icfp, galois connection, adjunction, category thoery, math
+
+One of my favorite papers at ICFP 2016 (in lovely
+[Nara, Japan](http://conf.researchr.org/home/icfp-2016)) was
+[Constructive Galois Connections ...](http://dl.acm.org/citation.cfm?id=2951934)
+by David Darais and David Van Horn.
+The central technical result is quite interesting, but a little
+intimidating, so I'd like to share a "de-generalization" of the result
+that I found helpful to understand.
+
+<!-- more -->
+
+# History
+The paper is about how to do verified static analysis while taking
+advantage of the calculational approach of
+[Abstract Interpretation](http://www.di.ens.fr/~cousot/COUSOTpapers/Marktoberdorf98.shtml).
+The problem is that the Galois connections people use for abstract
+domains are not always computable.
```

Comment:
I didn't really want to get too into the details of the paper, I think the paper is better written than my summary would be.

There is an example in the paper which is relations on a product and products of relations.